pkg/globalstate/globalstate.go

```diff
@@ -0,0 +1,11 @@
+package globalstate
+
+import (
+	"github.com/exoscale/egoscale"
+)
+
+var (
+	OutputFormat         string
+	GlobalEgoscaleClient *egoscale.Client
```

Comment:
I'm thinking `Global` keyword is unnecessary as the package name (`globalstate`) holds the same information. Just a thought, you can leave it as is.